app/webpacker/components/RegistrationsV2/Context/registration_context.jsx

```diff
@@ -0,0 +1,7 @@
+import { createContext } from 'react';
+
+export const RegistrationContext = createContext({
+  registration: null,
+  refetch: () => {},
```

Comment:
Not super happy about a method being part of a context, but this is part of a bigger discussion.